taskfiles/utils.yml

```diff
@@ -1,64 +1,77 @@
 version: "3"
 
+set: ["u", "pipefail"]
+shopt: ["globstar"]
+
 tasks:
   # ===
   # CHECKSUM UTILS
   # ===
 
-  # @param {string} DATA_DIR The directory to compute the checksum for.
+  # @param {string[]} DATA_PATTERNS Path wildcard patterns to compute the checksum for.
   # @param {string} OUTPUT_FILE
-  # @param {[]string} [EXCLUDE_PATHS] A list of paths, relative to `DATA_DIR`, to exclude from the
-  # checksum.
+  # @param {string[]} [EXCLUDE_PATTERNS] Path wildcard patterns, relative to any `DATA_PATTERNS`, to
+  # exclude from the checksum.
   compute-checksum:
-    desc: "Tries to compute a checksum for the given directory and output it to a file."
+    desc: "Tries to compute a checksum for the given paths and output it to a file."
     internal: true
+    label: "{{.TASK}}-{{.OUTPUT_FILE}}"
     silent: true
     requires:
-      vars: ["DATA_DIR", "OUTPUT_FILE"]
+      vars: ["DATA_PATTERNS", "OUTPUT_FILE"]
     cmds:
       - >-
         tar cf -
-        --directory "{{.DATA_DIR}}"
         --group 0
         --mtime "UTC 1970-01-01"
         --numeric-owner
         --owner 0
         --sort name
-        {{- range .EXCLUDE_PATHS}}
+        --no-anchored
+        --wildcards
+        {{- range .EXCLUDE_PATTERNS}}
         --exclude="{{.}}"
```

Comment:
Can we add a comment about why the exclude pattern needs quotes but the include pattern doesn't?